src/main/java/by/exadel/internship/dto/formDTO/FormRegisterDTO.java

```diff
@@ -2,14 +2,16 @@
 
 import by.exadel.internship.dto.TimeForCallDTO;
 import by.exadel.internship.dto.enums.EnglishLevel;
-import by.exadel.internship.entity.location.City;
-import by.exadel.internship.entity.location.Country;
+import by.exadel.internship.dto.locationDTO.CityDTO;
```

Comment:
remove please from all package name `DTO`
```suggestion
import by.exadel.internship.dto.location.CityDTO;
```

Reply:
Done